https://bugs.gentoo.org/907091
https://bugs.gentoo.org/927796

Remove off64_t, lseek64, add AC_SYS_LARGEFILE to configure.ac instead

--- a/include/PGFplatform.h
+++ b/include/PGFplatform.h
@@ -536,8 +536,8 @@
 			return NoError;
 		}
 	#else
-		off64_t ret;
-		if ((ret = lseek64(hFile, 0, SEEK_CUR)) == -1) {
+		off_t ret;
+		if ((ret = lseek(hFile, 0, SEEK_CUR)) == -1) {
 			return errno;
 		} else {
 			*pos = (UINT64)ret;
@@ -554,7 +554,7 @@
 			return NoError;
 		}
 	#else
-		if ((lseek64(hFile, (off64_t)posOff, posMode)) == -1) {
+		if ((lseek(hFile, (off_t)posOff, posMode)) == -1) {
 			return errno;
 		} else {
 			return NoError;
--- a/configure.ac
+++ b/configure.ac
@@ -31,6 +31,7 @@
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADERS( [config.h] )
 AC_PROG_CXX
+AC_SYS_LARGEFILE
 
 dnl Check for doxygen support
 AC_PATH_PROG([DOXYGEN], [doxygen])
